bans-core/src/main/resources/lang/messages_es.yml

```diff
@@ -75,57 +78,57 @@ lists:
       footer: '&7Página &e%PAGE%&7.||ttp:Click para la siguiente página||cmd:/libertybans warns %TARGET% %NEXTPAGE%'
 
     perPage: 10
-    usage: '&cUso: /warns &e<player> [página]'
+    usage: '&cUso: /warns &e<jugador> [página]'
 
   blame:
-    noPages: '&c&o%TARGET%&r&7 no castigo a nadie aún.'
-    maxPages: '&7Página &e%PAGE%&7 no existe.'
+    noPages: '&c&o%TARGET%&r&7 no ha sancionado a nadie aún.'
+    maxPages: '&7La página &e%PAGE%&7 no existe.'
     permission:
       command: '&7No puedes hacer esto.'
 
     layout:
       header:
-        - '&7[&eID&7] &e&oSujeto &r&8/ &7Tipo de castigo'
+        - '&7[&eID&7] &e&oSujeto &r&8/ &7Tipo de Sanción'
         - 'Razón &8/ &7Fecha de Inicio'
       body:
         - '&7[&e%ID%&7] &e&o%VICTIM% &r&8 / &7%TYPE%'
         - '%REASON% &8/ &7%START_DATE%'
       footer: '&7Página &e%PAGE%&7.||ttp:Click para la siguiente página||cmd:/libertybans warns %TARGET% %NEXTPAGE%'
 
     perPage: 10
-    usage: '&cUso: /blame &e<player> [page]'
+    usage: '&cUso: /blame &e<jugador> [página]'
 
   history:
-    noPages: '&c&o%TARGET%&r&7 no tiene un historial.'
-    maxPages: '&7Página &e%PAGE%&7 no existe.'
+    noPages: '&c&o%TARGET%&r&7 no tiene historial.'
+    maxPages: '&7La página &e%PAGE%&7 no existe.'
     permission:
       command: '&7No puedes ver el historial.'
 
     layout:
       header:
-        - '&7[&eID&7] &8/ &7Tipo de Castigo'
-        - 'Operador &8/ &7Raz´no &8/ &7Fecha de Inicio'
+        - '&7[&eID&7] &8/ &7Tipo de Sanción'
+        - 'Operador &8/ &7Razón &8/ &7Fecha de Inicio'
       body:
         - '&7[&e%ID%&7] / %TYPE%'
         - '%OPERATOR% &8/ &7%REASON% &8/ &7%START_DATE%'
       footer: '&7Página &e%PAGE%&7.||ttp:Click para la página siguiente||cmd:/libertybans history %TARGET% %NEXTPAGE%'
 
     perPage: 10
-    usage: '&cUso: /history &e<player> [page]'
+    usage: '&cUso: /history &e<jugador> [página]'
 
 
 all:
   prefix:
      # If enabled, all messages will be prefixed
     enable: true
      # The prefix to use
-    value: '&6&lBans &r&8»&7 '
+    value: '&6&lLibertyBans &r&8»&7 '
 
    # When issuing commands, if the specified player or IP was not found, what should the error message be?
   not-found:
-    playerOrAddress: '&c&o%TARGET%&r&7 no fue encontrado ni online ni offline y no tiene una ip válida.'
-    uuid: '&c&o%TARGET%&r&7no es un UUID válido.'
-    player: '&c&o%TARGET%&r&7 no fue encontrado ni onlnie ni offline.'
+    playerOrAddress: '&c&o%TARGET%&r&7 no fue encontrado online o offline y no es una IP válida.'
+    uuid: '&c&o%TARGET%&r&7no es una UUID válida.'
+    player: '&c&o%TARGET%&r&7 no fue encontrado online ni offline.'
```

Comment:
Is there a good translation of "online" and "offline"?

Reply:
`"online"` could mean `"en linea"`, `"activo"`, `"presente"` and `"offline"` could mean `"ausente"`, `"no presente"`.
In this case, I thought the best option would be to leave online/offline since the translation options would be: `"&c&o%TARGET%&r&7 no fue encontrado en linea o fuera de linea"`, which is very literal, not implied and overextended.

Maybe changing a little bit it could be translated in `"El jugador &c&o%TARGET%&r&7 no fue encontrado en linea o en la base de datos"`
Which refers to the plugin database, which does not exactly stick to the literal translation, but gives the same meaning to express that the player is not present on the server.

Reply:
I like your translation referring to the database much better. That _is_ what I meant to convey when I wrote the original message in English. Good judgment.

So, without further ado, seeing as you've committed the change already, we'll go with it.